### ISSUE TITLE
CLI command doesn't work

### ISSUE THREAD

**Issue by rahatarmanahmed** [GochoMugo/json-concat#2]
``` sh
➜  ~ npm i -g json-concat
/Users/rahat/.nvm/versions/node/v4.3.1/bin/json-concat -> /Users/rahat/.nvm/versions/node/v4.3.1/lib/node_modules/json-concat/cmd.js
/Users/rahat/.nvm/versions/node/v4.3.1/lib
└── json-concat@0.0.0

➜  ~ json-concat
/Users/rahat/.nvm/versions/node/v4.3.1/bin/json-concat: line 1: syntax error near unexpected token `('
/Users/rahat/.nvm/versions/node/v4.3.1/bin/json-concat: line 1: `(function() {'
```

**Comment by GochoMugo** [comment]
Ooh, I forgot to add a shebang! Fixing this asap!

**Comment by GochoMugo** [comment]
Quick, dirty fix, before I get time to work on this:

```
$ echo '#!/usr/bin/env node' | cat - $(which json-concat) > tmp
$ chmod +x tmp
$ mv tmp `readlink -f $(which json-concat)`
```

**Comment by GochoMugo** [comment]
**FIXED IN v0.0.1**
